config/initializers/colorful_logger.rb

```diff
@@ -21,6 +21,19 @@ def call(severity, _time, _progname, msg)
     formatted_severity = sprintf('%-3s', SEVERITY_TO_TAG_MAP[severity])
     color = SEVERITY_TO_COLOR_MAP[severity]
```

Comment:
I originally erased the whole file, but many logs were still not in JSON format.  I removed the coloring and custom log levels from the file and renamed it JSON logger.  This allows it to put the rest of the logs in JSON format.